packages/code-snippet-block/src/components/CodeMirrorEditor/themes/defaultTheme.ts

```diff
@@ -0,0 +1,17 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```

Comment:
maybe also rename it to just `default` as it is already in the themes folder. same as you did with other themes like `cobalt`